governance/polls/Add GUNIV3DAIUSDC2-A with 0.01% Fee Tier as a new Vault Type - November 29, 2021.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Add GUNIV3DAIUSDC2-A with 0.01% Fee Tier as a new Vault Type - November 29, 2021
```

Comment:
I don't think you need "with 0.01% Fee Tier" here, or in the file name

Reply:
fxd!